apigee_sync.go

```diff
@@ -160,6 +161,13 @@ func pollChangeAgent() error {
 			return err
 		}
 
+		if changesRequireDDLSync(resp) {
+			log.Info("Detected DDL changes, going to fetch a new snapshot to sync...")
+			return changeServerError{
```

Comment:
I would argue that it is an error, since we have detected a change which we cannot process.  I think it's an error in the sense that it would be an error to continue to process changes when we don't have the schema for it yet.